oops/backplane/unittester_backplanes_log_diff.py

```diff
@@ -1,112 +0,0 @@
-#!/usr/bin/env python3
```

Comment:
I am quite happy to see this log differencer go away. However, rather than delete it entirely, please just move it to the ideas subdirectory. I would not want to start writing it again from scratch if we ever need something like it in the future.